src/components/chip-group/chip-group.tsx

```diff
@@ -193,6 +193,7 @@ export class ChipGroup implements InteractiveComponent {
       el.interactive = true;
       el.scale = this.scale;
       el.selectionMode = this.selectionMode;
+      el.setAttribute("aria-labelledby", this.label);
```

Comment:
Shouldn't this just be `aria-label`? I thought labelledby takes an `id`?

Reply:
@geospatialem can you advise here? Is this not needed at all?